modal/_container_io_manager.py

```diff
@@ -677,11 +683,15 @@ async def handle_input_exception(
                 data_format=api_pb2.DATA_FORMAT_PICKLE,
                 results=results,
             )
-            await self.exit_context(started_at)
+            await self.exit_context(started_at, io_context.input_ids)
 
-    async def exit_context(self, started_at):
+    async def exit_context(self, started_at, input_ids: List[str]):
         self.total_user_time += time.time() - started_at
         self.calls_completed += 1
+        self.current_input = None
```

Comment:
`self.current_input_id` here?

Reply:
Ah thanks for catching! I'll update the test to check this too (although I suspect we'll get rid of the singular `current_input_id` state soon since it's not compatible with concurrent inputs or batching)

Reply:
Actually I'm removing this line altogether as I'm not sure if its safe/good to reset current_input_id between inputs (in case it's used to associate logs etc. to inputs)